eng/Packages.Data.props

```diff
@@ -103,6 +103,12 @@
     <PackageReference Update="System.ValueTuple" Version="4.5.0" />
     <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
     <PackageReference Update="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Update="Microsoft.Extensions.Configuration" Version="2.1.0" />
```

Comment:
These may be limiting in functionality.  I'm going to upgrade to see if we can get the features we expect in the latest DI packages and if there are reasons we can't upgrade the repo to these, we can look at polyfill.